contributing/releasing.md

```diff
@@ -54,8 +54,13 @@ If you need to cut a hotfix release, run the following command instead:
     scripts/release cut --hotfix
 
 A hotfix release is like a regular release, but its scope is limited specifically to the fix. Hotfix
-release candidates start from origin/stable rather than origin/develop, but otherwise the entire
-process is the same.
+release candidates start from origin/stable rather than origin/develop. 
```

Comment:
Please consider adding a subheading to separate the Hotfix instructions from the rest of the doc so it can be easily found.